.github/workflows/examples-tizen.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: connectedhomeip/chip-build-tizen:0.5.79
+            image: connectedhomeip/chip-build-tizen:0.5.81
```

Comment:
I believe updates should generally be done globally (change all workflows).
This makes it easier to ensure that images are consistent (and that chip-build-vscode we use can compile everything)

Reply:
OK, so this PR can wait for #19937 to get merged

Reply:
We seem to have trouble with Infineon image builds. Resolving this comment -we can move everything in sync once build issues are resolved.